cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -490,18 +481,13 @@ public void getProgramRuntimeArgs(HttpRequest request, HttpResponder responder,
 
     Id.Program id = Id.Program.from(namespaceId, appId, type, programId);
 
-    try {
-      if (!store.programExists(id)) {
-        responder.sendString(HttpResponseStatus.NOT_FOUND, "Program not found");
-        return;
-      }
-      Map<String, String> runtimeArgs = preferencesStore.getProperties(id.getNamespaceId(), appId,
-                                                                       programType, programId);
-      responder.sendJson(HttpResponseStatus.OK, runtimeArgs);
-    } catch (Throwable e) {
-      LOG.error("Error getting runtime args {}", e.getMessage(), e);
-      responder.sendStatus(HttpResponseStatus.INTERNAL_SERVER_ERROR);
+    if (!store.programExists(id)) {
+      responder.sendString(HttpResponseStatus.NOT_FOUND, "Program not found");
```

Comment:
I don't want to change any non-``INTERNAL_SERVER_ERROR`` responses in this PR. Is that ok?